pytorch_lightning/trainer/training_loop.py

```diff
@@ -335,7 +335,9 @@ def train(self):
         # if reload_dataloaders_every_epoch, this is moved to the epoch loop
         if not self.reload_dataloaders_every_epoch:
             self.reset_train_dataloader(model)
-        self.reset_val_dataloader(model)
+
+        if model.val_dataloader is not None:
+            self.reset_val_dataloader(model)
```

Comment:
it looks like val dataloaders get reloaded every epoch no matter the setting, `self.reload_dataloaders_every_epoch`. Isn't this a bug?

Reply:
that’s how it was before no? i think that was intended this way

Reply:
is there an issue about it?

Reply:
if it's intended, ok. Just the name of the flag is a bit misleading then :)

> is there an issue about it?

There does not seem to be one.